.nvmrc

```diff
@@ -0,0 +1 @@
+v16.17.0
```

Comment:
Any reason to force v16 here?

Reply:
Just to keep dev env consistent across the repos. Updated the same for Snapshot UI repo as well.